php/tests.php

```diff
@@ -41,4 +41,58 @@ public function testAddParagraphsMalformed(){
         addParagraphs($input);
     }
 
+
```

Comment:
Please add the tests to a tests folder and rename this!

Reply:
wohooooo final comment! done it